src/_common/comment/add/add.vue

```diff
@@ -25,24 +20,35 @@
 			<app-form-control-errors label="comment" />
 		</app-form-group>
 
-		<div class="text-right">
+		<p v-if="shouldShowGuidelines" class="-guidelines">
+			Remember to be respectful and follow our
```

Comment:
Couldn't translate because it contains a vue component.

Reply:
Bah and I butchered my community activity. Why can't I just do this there D: